neuraxle/runners.py

```diff
@@ -257,7 +258,7 @@ def load_checkpoint_recursive(self, steps_as_tuple, data_inputs, expected_output
 
         return None, None
 
-    def create_checkpoint_names(self, steps_as_tuple):
+    def create_checkpoint_names(self, steps_as_tuple: List[Tuple[str, BaseStep]]):
```

Comment:
@alexbrillant  I feel that this has to do with the existing type `NamedTupleList` which can be imported and is defined in `base.py`. 

But this will change anyways when fixing the TDA: the `create_checkpoint_names` might move into Pipeline or TruncableSteps or MetaStepsMixin or something like that and will not need such an argument (because the `steps_as_tuple` will be in `self` at that moment so it won't be passed as argument).